web-next/app/sdd/ai-spec-driven-development-guide/page.module.css

```
@@ -20,9 +20,10 @@
   --color-border-secondary: #1e3049;
   --color-border-tertiary: #16283d;
 
-  --font-sans: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Hiragino Kaku Gothic ProN',
-    'Noto Sans JP', 'Yu Gothic', sans-serif;
-  --font-mono: var(--font-mono, 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, monospace);
+  --font-sans:
+    -apple-system, BlinkMacSystemFont, "Segoe UI", "Hiragino Kaku Gothic ProN", "Noto Sans JP",
```

Stylelint (17.14.1)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)